osu.Game/Overlays/Changelog/ChangelogBuild.cs

```diff
@@ -76,11 +76,11 @@ private void load(OsuColour colours)
 
                     var entryColour = entry.Major ? colours.YellowLight : Color4.White;
 
-                    title.AddIcon(FontAwesome.Solid.Check, t =>
+                    title.AddIcon(entry.Type == ChangelogEntryType.Fix ? FontAwesome.Solid.Check : FontAwesome.Solid.Plus, t =>
                     {
                         t.Font = fontSmall;
                         t.Colour = entryColour;
-                        t.Padding = new MarginPadding { Left = -17, Right = 5 };
+                        t.Padding = new MarginPadding { Left = -17, Top = 5 };
```

Comment:
can't this be done by positioning the icon at the left-center instead of adjusting the padding?
```csharp
t.Anchor = Anchor.CentreLeft;
t.Origin = Anchor.CentreLeft;
```

Reply:
Yep, These should hopefully not be necessary any more.

Reply:
I tried. Not working as expected:
![Screenshot_2](https://user-images.githubusercontent.com/22874522/65270246-6e288b80-db23-11e9-8ba9-d8afc47ee10e.png)

Reply:
Seems correct to me, we're talking about the vertical padding, you should adjust the horizontal padding yourself (i think setting `Origin` to `Anchor.CentreRight` will push it out to the left).

Reply:
Nope, that's not happening

Reply:
As in try:

```
Anchor = CentreLeft,
Origin = CentreLeft,
Padding = { Right = 5 }
```

Reply:
Still no

Reply:
This works for me:

```csharp
title.AddIcon(entry.Type == ChangelogEntryType.Fix ? FontAwesome.Solid.Check : FontAwesome.Solid.Plus, t =>
{
    // ...
    t.Padding = new MarginPadding { Left = -4, Right = 5 };
    t.Anchor = Anchor.CentreRight;
    t.Origin = Anchor.CentreRight;
});

title.AddText(entry.Title, t =>
{
    // ...
    t.Anchor = Anchor.CentreLeft;
    t.Origin = Anchor.CentreLeft;
});
```

Result (with crude visual explanation - circles are anchor points):
![image](https://user-images.githubusercontent.com/20418176/65317636-825c9f00-db9c-11e9-8994-377c77d1cff0.png)

I would have added the padding to the entry title text but it seems to affect spacing for some reason. Will recheck framework-side and open an issue where appropriate.

Reply:
Ok, doing the padding on the title text affects spacing because it's in a text flow, so it's *not* a bug; if we wanted to set it there then this would need some container refactoring.

Reply:
> Ok, doing the padding on the title text affects spacing because it's in a text flow, so it's _not_ a bug; if we wanted to set it there then this would need some container refactoring.

It deserves an individual pull request then